src/hapi/pipelines/database/operational_presence.py

```diff
@@ -168,37 +161,36 @@ def populate(self, debug=False):
                             logger.error(f"Sector {sector_orig} not in table")
                             continue
 
-                        org_ref = self._org.data[
-                            (
-                                clean_name(org_acronym).upper(),
-                                clean_name(org_name),
-                                org_type_code,
-                            )
-                        ]
                         admin2_ref = self._admins.admin2_data[admin2_code]
-                        row = (resource_id, org_ref, sector_code, admin2_ref)
+                        row = (
+                            resource_id,
+                            admin2_ref,
+                            org_acronym,
+                            org_name,
+                            sector_code,
+                        )
                         if row in rows:
                             number_duplicates += 1
                             continue
                         rows.append(row)
                         operational_presence_row = DBOperationalPresence(
                             resource_hdx_id=resource_id,
                             admin2_ref=admin2_ref,
-                            org_ref=org_ref,
+                            org_acronym=org_acronym,
+                            org_name=org_name,
                             sector_code=sector_code,
                             reference_period_start=time_period_start,
-                            # TODO: Add to scraper (HAPI-199)
-                            source_data="not yet implemented",
                         )
                         self._session.add(operational_presence_row)
+                        # TODO: move this commit out of the loop once you figure out why it needs to be here
+                        self._session.commit()
```

Comment:
What if you run it each time you add an entry to the org table? Or did you get a chance to try your own suggestion - populating orgs all at once with a dict? 

(If you want to do bulk updates - [here](https://github.com/OCHA-DAP/hdx-scraper-who/blob/feature/who.py#L251-L281) is some code I wrote recently to do so)

Reply:
I didn't get a chance to try populating all of the orgs at once, but that's next!  I did try running each time I added a new org, but the issue was still there.  It's almost like it's trying to commit the operational presence rows to the org table.